fds/services/git_service.py

```diff
@@ -55,3 +55,9 @@ def add(self, add_argument: str) -> Any:
 
     def commit(self, message: str) -> Any:
         execute_command(["git", "commit", "-am", message], capture_output=False)
+
+    def clone(self, url: str) -> Any:
+        execute_command(["git", "clone", url], capture_output=False)
+        # Todo: Also support getting the directory to clone from the user
```

Comment:
Yes, should be easy to do as a 2nd positional argument

Reply:
done